packages/core/src/utils/breakpoints.ts

```diff
@@ -1,6 +1,7 @@
 export const breakpoints = {
-    small: 768,
-    medium: 992,
+    /** @deprecated `small` og `medium` er identiske. En `small device` går fra 0 til `medium - 1`. */
+    small: 680,
```

Comment:
Hvorfor beholde denne?

Reply:
Tenkte spare en breaking change

Reply:
Men vil det ikke bli breaking uansett fordi small/medium endrer seg? Regner med at de fleste må gå gjennom alle løsningene sine for å sjekke

Reply:
Kanskje. Jeg har tenkt at layouts som er "ca pixel perfect" med tanke på breakpoints har gjort noe custom, ikke brukt disse variablene og hjelpefunksjonene. Sånn at en tweaking av verdiene og hjelpefunksjonene bør være ganske safe.

Er åpen for å gjøre denne til breaking. I så fall setter jeg pull requesten i kø bak IE-opprydding (eventuelt samles i samme release). Uansett ingen hast :)